docs/transition.md

```diff
@@ -50,25 +61,26 @@ continue transferring data, you will no longer be able to run jobs or create new
 ## Onboarding Live Sessions
 
 To ensure a smooth transition from the Shared Cluster to Unity, we will hold two online 
```

Comment:
Should be "held", not "will hold", now that these events have already happened.

Reply:
Old comment not about this change: I think it's only UML PI's that will get sufficient space in /project temporarily... right?
The recording links worked for me without being logged in.